app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Salaried teacher training"
+description: |-
+    Explore salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
+related_content:
+    Nathan's experience of salaried teacher training: "/blog/salaried-teacher-training-classroom-learning"
+    If you've worked as an unqualified teacher: "/train-to-be-a-teacher/assessment-only-route-to-qts"
+    What to expect on your teacher training: "/blog/what-to-expect-on-your-teacher-training"
+promo_content:
+    - content/train-to-be-a-teacher/promos/find-your-course
+    - content/funding-and-support/promos/get-adviser-funding-promo
+navigation: 20.20
+navigation_description: Find out about salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
```

Comment:
```suggestion
navigation_description: Find out about salaried courses, including School Direct (salaried), postgraduate teaching apprenticeships and Teach First.
```

This is pretty long compared to our other nav descriptions - not sure whether my suggestion makes sense though!

Reply:
will tweak the first half, would probs be keen to keep the 'delivered by Teach First' bit though just to make sure we're 100% saying the right thing, policy wise